docs/v3.x/guides/update-version.md

```diff
@@ -58,7 +58,13 @@ For example moving from `3.0.4` to `3.0.5`
 
 ::::
 
-Then run either `yarn install` or `npm install` to install the specified version.
+Install the specified version:
```

Comment:
Can you use the Vuepress tabs here? Just to keep it in line with other instances where we reference yarn vs npm.

see: https://www.npmjs.com/package/vuepress-plugin-tabs#usage

Reply:
@derrickmehaffy Just updated it.